osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -40,7 +40,7 @@ public class OsuSliderBar<T> : SliderBar<T>, IHasTooltip, IHasAccentColour
         private readonly Box rightBox;
         private readonly Container nubContainer;
 
-        public virtual LocalisableString TooltipText { get; private set; }
+        public virtual LocalisableString TooltipText { get; protected set; }
```

Comment:
Seems no longer necessary.
```suggestion
        public virtual LocalisableString TooltipText { get; private set; }
```